libs/wingsdk/src/target-tf-aws/counter.inflight.ts

```diff
@@ -40,6 +40,10 @@ export class CounterClient implements ICounterClient {
     return parseInt(newValue) - amount;
   }
 
+  public async dec(amount = 1): Promise<number> {
```

Comment:
Perhaps we can put this in `CounterBase`? Seems like there's no need to implement this for every target?

Reply:
Great idea, I'll change and do some tests